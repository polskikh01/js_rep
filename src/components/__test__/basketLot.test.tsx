import React from 'react';
import { mount } from 'enzyme';
import { describe, it, expect } from '@jest/globals';

import BasketLot from '../basketLot';

import img from '../../asset/img/basket.png';

describe('<BasketLot/>', () => {
    const title = 'PRODUCT';
    const price = 'OOooOOooOoOo';
    it('Отрисовывется без ошибок', () => {
        const wrapper = mount(<BasketLot img={img} title={title} price={price} />);

        expect(wrapper).toMatchSnapshot();
    });
});
