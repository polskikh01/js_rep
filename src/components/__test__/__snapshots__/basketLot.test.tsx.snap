// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BasketLot/> Отрисовывется без ошибок 1`] = `
<BasketLot
  img="FILE_MOCK"
  price="OOooOOooOoOo"
  title="PRODUCT"
>
  <div
    className="basketLot"
  >
    <div
      className="lot"
    >
      <img
        src="FILE_MOCK"
      />
      <span>
        PRODUCT
      </span>
      <span>
        OOooOOooOoOo
        ₽
      </span>
    </div>
  </div>
</BasketLot>
`;
